docarray/array/mixins/io/common.py

```diff
@@ -1,4 +1,4 @@
-from typing import Union, TextIO, BinaryIO, TYPE_CHECKING, Type
+from typing import Union, TextIO, BinaryIO, TYPE_CHECKING, Type, Optional
```

Comment:
Optional is not used
```suggestion
from typing import Union, TextIO, BinaryIO, TYPE_CHECKING, Type
```